### PR TITLE
Mannequin form frontend

### PR DESCRIPTION
**Pull request by jbernardoviana**
Hey! just did an orgasmic mannequin form, hope you love it

### CODE REVIEW

File: app/views/mannequins/_form.html.erb [hunk 1]
```diff
@@ -10,21 +11,25 @@
       <%= m.input :age, collection: 18..60, prompt: "Select your age" %>
       <%= m.input :category, collection: ["Male", "Female"] %>
```
Comment (cosydney) [inline app/views/mannequins/_form.html.erb]: category = %w(beaute haute-couture atypique new-face hipster
)

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -13,3 +13,4 @@
 @import "components/index";
 @import "pages/index";
 @import "vendor/index";
+@import "mannequins"
```
Comment (cosydney) [inline app/assets/stylesheets/application.scss]: don't forget the ; at the end

File: app/assets/stylesheets/components/_button.scss [hunk 3]
```diff
@@ -37,3 +37,95 @@
   }
 }
 
+.btn-medium {
+  color: #3B3B3C;
```
Comment (cosydney) [inline app/assets/stylesheets/components/_button.scss:56]: Use $brand-color variable unstead of #3B3B3C

File: app/assets/stylesheets/components/_button.scss [hunk 4]
```diff
@@ -37,3 +37,95 @@
...
+  border: 1px solid #999999;
+  padding: 10px 15px;
+  border-radius: 50px;
+  font-weight: lighter;
+  opacity: 0.5;
+  background-color: lightblue;
```
Comment (cosydney) [inline app/assets/stylesheets/components/_button.scss:62]: Can we set the new $blue to the lightblue so we can change it all once if needed ?

File: app/assets/stylesheets/components/_button.scss [hunk 5]
```diff
@@ -37,3 +37,95 @@
...
+}
+
+.btn-medium:hover {
+  opacity: 0.8;
+  text-decoration: none;
+  color: #3B3B3C;
```
Comment (cosydney) [inline app/assets/stylesheets/components/_button.scss:68]: Let's set up variable for this color

File: app/assets/stylesheets/mannequins.scss [hunk 6]
```diff
@@ -1,3 +1,34 @@
...
+  padding: 40px;
+  color: #3B3B3C;
+  box-shadow: 1px 2px 5px 0px rgba(122,115,122,1);;
+  background-color: #F0F1F1;
+  /*font-family: */
+}
+
+/*top of each pannel: */
+.panel-heading {
+  color:#3B3B3C !important;
```
Comment (cosydney) [inline app/assets/stylesheets/mannequins.scss:16]: Lot's of important is that normal ?

File: app/views/mannequins/_form.html.erb [hunk 7]
```diff
@@ -10,21 +11,25 @@
...
       <%= m.input :description, as: :text %>
-      <%= m.input :height %>
+      <%= m.input :height, hint: 'You cannot change your username.' %>
       <%= m.input :waist %>
       <%= m.input :chest %>
       <%= m.input :hips %>
       <%= m.input :hair_color %>
```
Comment (cosydney) [inline app/views/mannequins/_form.html.erb]: we need to add gender : men or women.. I've added the table column in my code

File: app/views/mannequins/edit.html.erb [hunk 8]
```diff
@@ -1,25 +1,59 @@
...
-    </div>
-  </div>
-  <div class="row">
-    <div class="col-xs-6">
-      <div class="text-center">
-        <%= render 'form' %>
+<div id="accordion" role="tablist" aria-multiselectable="true">
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-xs-2 col-xs-offset-5 text-center">
```
Comment (cosydney) [inline app/views/mannequins/edit.html.erb:6]: cool mobile view woop woop 👍

File: app/views/mannequins/_form.html.erb [hunk 9]
```diff
@@ -10,21 +11,25 @@
       <%= m.input :age, collection: 18..60, prompt: "Select your age" %>
       <%= m.input :category, collection: ["Male", "Female"] %>
       <%= m.input :description, as: :text %>
```
Comment (cosydney) [inline app/views/mannequins/_form.html.erb]: Could we possibly make the description to be only 160 characters ?

**Comment by cosydney** [comment]
Cool man !! Things i've noticed: 
- Once I submited my info i get redirected to my profile
- Same for photos. Could we be redirected to the edit page unstead? 
- Do we need the back button? it goes the a weird looking index.. (i think i've updated that in my code)

Could we make the whole area clickable for : 

capture d ecran 2016-08-11 18 22 03

I've noticed you are mentionning my name, would be cool to have more information like: 

Hi Sydney this is your dashboard where you can update your photos your measurements see your profile, just click on the relevant button to do that :)
- date: 

capture d ecran 2016-08-11 18 26 30

File: app/assets/stylesheets/mannequins.scss (hunk 6 above)
Comment (jbernardoviana) [inline app/assets/stylesheets/mannequins.scss:16]: it's to override bootstrap default mate

**Comment by jbernardoviana** [comment]
Form finnaly perfect

**Pull request merged**